### PR TITLE
added ejplugins recipe

### PR DESCRIPTION
**Pull request by chrisjsewell**
python noarch, built successfully on local machine

### CODE REVIEW

**Comment by chrisjsewell** [comment]
win_64 failure is a known issue: #7715

**Comment by chrisjsewell** [comment]
hey @conda-forge/help-python ready for review and merge

File: recipes/ejplugins/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - ejplugins.schema
+    - ejplugins.test_files
+    - ejplugins.utils
+
+about:
+  home: The package home page
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: parser plugins for jsonextended
```
Comment (chrisburr) [inline recipes/ejplugins/meta.yaml]:
```suggestion
  summary: Parser plugins for the jsonextended package and validation schema, to convert output files from materials simulation packages to a JSON format.
```

File: recipes/ejplugins/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - ejplugins.crystal
+    - ejplugins.gulp
+    - ejplugins.lammps
+    - ejplugins.qespresso
+    - ejplugins.schema
+    - ejplugins.test_files
+    - ejplugins.utils
+
+about:
+  home: The package home page
```
Comment (chrisburr) [inline recipes/ejplugins/meta.yaml]:
```suggestion
  home: https://github.com/chrisjsewell/ejplugins
```

**Pull request merged**